receipt_langsmith/tests/spark/test_evaluator_journey_viz_cache.py

```
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+import json
+import os
+
+import pytest
+
+from receipt_langsmith.spark.evaluator_journey_viz_cache import (
+    build_journey_cache,
+)
+
+PARQUET_DIR = "/tmp/langsmith-traces/"
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/langsmith-traces/"

(S108)